sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractKeyPhrasesResultCollectionDebugView.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Debugger Proxy class for <see cref="ExtractKeyPhrasesResultCollection"/>.
+    /// </summary>
+    public class ExtractKeyPhrasesResultCollectionDebugView
+    {
+        private ExtractKeyPhrasesResultCollection BaseCollection { get; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractKeyPhrasesResultCollectionDebugView"/> class.
+        /// </summary>
+        /// <param name="collection"></param>
+        public ExtractKeyPhrasesResultCollectionDebugView(ExtractKeyPhrasesResultCollection collection)
+        {
+            BaseCollection = collection;
+        }
+
+        /// <summary>
+        /// Collection of <see cref="ExtractKeyPhrasesResult"/> objects corresponding
+        /// to a batch of documents.
+        /// </summary>
+        public List<ExtractKeyPhrasesResult> Items
```

Comment:
my concern with having `Items` is that the user might see this in the debugger and then they will expect for `ExtractKeyPhrasesResultCollection` to have a property called `Items`.

Reply:
Add `[DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]` on this property.